src/Styleguide/Pages/Artist/routes.tsx

```diff
@@ -18,6 +30,16 @@ export const routes = [
       {
         path: "cv",
         Component: CVRoute,
+        query: CVQuery,
+        prepareVariables: params => ({
+          artistID: "pablo-picasso",
+          first: 10,
+          at_a_fair: false,
+          solo_show: true,
+          sort: "start_at_desc",
+          is_reference: true,
+          visible_to_public: false,
+        }),
```

Comment:
@mzikherman - this is a good example of how we can leverage normal relay stuff alongside the router -- like hoist one query up and get some initial data ready to render, and then all of the other CV items, wrapped in QueryRenderers, can cascade in in time down the page.

Reply:
So how many query renderers will there be in a rendered tree? Ideally there’d only be 1, right?

Reply:
Well, I've been adding them for each of these components, in order to be able to fetch and render them.

I'm not sure how that's supposed to look when we put it all together, like one root query renderer that includes them or something?

Reply:
It seems straight forward enough to combine them into one (or whatever we decide on), they all take an `artistID` and spread their own fragments/etc (all on `Artist`), so should be v possible w minimal refactoring of the existing structure.

Reply:
Via found-relay we don't need QueryRenderer's, but they've been nice process step in terms of getting stuff rendering and then moving things into the proper routing structure. If we can get rid of them and consolidate all of our queries here then that would be ideal.

Reply:
> Via found-relay we don't need QueryRenderer's

Yeah sorry, I meant `fetchQuery` which `QueryRenderer` wraps and presumably `found-relay` wraps. In other words, a single query and fetch, rather than multiple.

> If we can get rid of them and consolidate all of our queries here then that would be ideal, which is the plan.

Perfect, that’s what I wanted to know 👍